### ISSUE TITLE
diagnosticLevel can't be overwritten for a specific file

### ISSUE THREAD

**Issue by icodeyou** [streetsidesoftware/vscode-spell-checker#2595]
Hello,

I have these settings for CSpell : 
```json
"cSpell.diagnosticLevel": "Warning",
    "cSpell.overrides": [
        {
            "language": "fr",
            "filename": "**/*_fr.yaml",
            "diagnosticLevel": "Hint" // Warning here
        }
    ],
```
I can overwrite `"language"` and `"filename"`, but not `"diagnosticLevel"`. If I do this, I have a warning. However, this warning has no reason to be, because it's working on my project (I have a `diagnosticLevel` of type `"Warning"` on all my files except for the one that I overrides)

<img width="384" alt="image" src="https://user-images.githubusercontent.com/15326889/225690898-5002ec90-347c-467c-9796-d21c52cf12e6.png">

**Comment by Jason3S** [comment]
@icodeyou,

It is not currently supported to use `overrides` to set `diagnosticLevel`.

You can use a [Multi-Root Workspace](https://code.visualstudio.com/docs/editor/multi-root-workspaces).

<img width="1149" alt="image" src="https://user-images.githubusercontent.com/3740137/225705929-c3aea184-c6ef-4a78-b78e-76a129f4505e.png">

**Comment by icodeyou** [comment]
A Multi-Root workspace is not a solution for me because I want to apply this to all files `**/*_fr.yaml` as I wrote in the code above.

I don't understand why you say that it is not supported, because the solution is working on my computer. It clearly overrides the property "diagnosticLevel"

I don't understand why there is a warning if it is working.
